index.bs

```diff
@@ -706,6 +733,17 @@ To <dfn>trigger attribution</dfn> given an [=attribution trigger=] |trigger| run
 1. Increment |sourceToAttribute|'s [=attribution source/number of reports=] value by 1.
 1. If |trigger|'s [=attribution trigger/dedup key=] is not null, [=list/append=] it to |sourceToAttribute|'s
     [=attribution source/dedup keys=].
+1. Let |rateLimitRecord| be a new [=attribution rate-limit record=] with the items:
+
+    : [=attribution rate-limit record/source site=]
+    :: |sourceSite|
+    : [=attribution rate-limit record/attribution destination=]
+    :: |attributionDestination|
+    : [=attribution rate-limit record/source type=]
+    :: |sourceToAttribute|'s [=attribution source/source type=]
+    : [=attribution rate-limit record/trigger time=]
+    :: |trigger|'s [=attribution trigger/trigger time=]
+1. Add |rateLimitRecord| to the [=attribution rate-limit cache=].
```

Comment:
Do we want to note removal of these entries somewhere? E.g. the user agent may remove all record whose trigger time is less than now - cutoff